### ISSUE TITLE
Selected items in parent

### ISSUE THREAD

**Issue by patrick-mackay** [braheem/angular-bootstrap-select#1]
Hi,

I'm trying to use this component but I'm having a problem that I've been unable to fix yet. When checking the list of selected items in the parent component (app.component in your demo), the list shows the initials values, and is not updated when some selections are made.

add this in app.component.html and you'll see what I mean.

```
<ul *ngIf="myCitiesString && myCitiesString.length > 0">
  <li *ngFor="let selection of myCitiesString">{{selection}}</li>
</ul>
```

the "same" code inside city-multiselect-string works perfectly, but the parent is not getting the list refreshed when selections are made.

can you point me in the right direction to fix this? I can guess that some emitter is missing, but I don't have the knowledge to know where and how.

Thanks

**Comment by braheem** [comment]
Thanks for catching this. It was a simple fix, just needed to emit changes from the mySelectionsChange EventEmitter to notify the parent component.

Should work now, take a look at the recent commits.

**Comment by patrick-mackay** [comment]
Thanks Abraham. I'll check it tomorrow and let you know.

I'm sorry for not responding on time, but github didn't notice me about your answer. It only did when you closed it.

Patrick.

On 26-02-2018 22:21, Abraham Al-Dabbagh wrote:

Closed #1<https://github.com/braheem/angular-bootstrap-select/issues/1>.

**Comment by patrick-mackay** [comment]
Thanks for the fix Abraham.
